### PR TITLE
feat: #WB2-850, add resource copy

### PR DESCRIPTION
**Pull request by david-cc**
# Description

Add resource copy.

After selecting a resource, through the actionbar a user can now copy a resource.

https://edifice-community.atlassian.net/browse/WB2-850

## Type of change

Please check options that are relevant.

- [ ] Chore (PATCH)
- [ ] Doc (PATCH)
- [ ] Bug fix (PATCH)
- [X] New feature (MINOR)
- [ ] Breaking change (MAJOR)

> PATCH: refactor, internal or non-breaking change which fixes an issue
>
> MINOR: non-breaking change which adds functionality
>
> MAJOR: fix or feature that would cause existing functionality to not work as expected

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings

### CODE REVIEW

File: frontend/src/services/api/index.ts [hunk 1]
```diff
@@ -166,6 +166,16 @@ export const restoreAll = async ({
     .restoreAll(trashParameters, useAssetIds);
 };
 
+export const copyResource = async (
```
Comment (ccreusat) [inline frontend/src/services/api/index.ts:169]: @david-cc pense à mettre de la JsDoc stp

File: frontend/src/services/queries/index.ts [hunk 2]
```diff
@@ -377,6 +378,82 @@ export const useDelete = () => {
   });
 };
 
+export const useCopyResource = () => {
```
Comment (ccreusat) [inline frontend/src/services/queries/index.ts:381]: @david-cc pareil, un petit commentaire de jsdoc comme les autres 🙏🏼

File: frontend/src/services/api/index.ts (hunk 1 above)
Comment (david-cc) [reply frontend/src/services/api/index.ts:175]: jsdoc ajoutée

File: frontend/src/services/queries/index.ts (hunk 2 above)
Comment (david-cc) [reply frontend/src/services/queries/index.ts:385]: jsdoc ajoutée

**Pull request merged**